<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Evan Teran</author>
 <class>BinarySearcherPlugin::DialogBinaryString</class>
 <widget class="QDialog" name="BinarySearcherPlugin::DialogBinaryString">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Binary String</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="BinaryString" name="binaryString">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="chkSkipNoAccess">
     <property name="text">
      <string>Skip Regions With No Access Rights</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="chkCaseSensitive">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Case Sensitive</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="chkAlignment">
     <property name="text">
      <string>Show Results With This Address Alignment</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="cmbAlignment">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>2 Byte Alignment</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4 Byte Alignment</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8 Byte Alignment</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QProgressBar" name="progressBar"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BinaryString</class>
   <extends>QFrame</extends>
   <header>BinaryString.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>chkSkipNoAccess</tabstop>
  <tabstop>chkCaseSensitive</tabstop>
  <tabstop>chkAlignment</tabstop>
  <tabstop>cmbAlignment</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BinarySearcherPlugin::DialogBinaryString</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>417</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>458</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BinarySearcherPlugin::DialogBinaryString</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>443</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
